docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+    "        \n",
+    "- `labels` is a list of lists, such that `labels[i]` is a list of token labels of the `i`'th sentence. Same format requirements as `cleanlab.rank.get_label_quality_scores`. \n",
+    "\n",
+    "Here, indicies are a tuple `(i, j)` unless otherwise specified, which refers to the `j`'th word-level token of the `i`'th sentence. Given that each sentence has different number of tokens, we store `pred_probs` and `labels` as `.npz` files, which can be easily converted to dictionaries. Use `read_npz` to retrieve `pred_probs` and `labels` in nested-list format. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "519cb80c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels = read_npz('labels.npz') \n",
+    "pred_probs = read_npz('pred_probs.npz') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dc3150f",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find potential label issues \n",
+    "\n",
+    "Based on the given labels and out-of-sample predicted probabilities, cleanlab can quickly help us identify label issues in our dataset. Here we request that the indices of the identified label issues be sorted by cleanlab’s self-confidence score, which measures the quality of each given label via the probability assigned to it in our model’s prediction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "95dc7268",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cleanlab found 2255 potential label issues. \n",
+      "The top 20 most likely label errors:\n",
+      "[(2907, 0), (19392, 0), (9962, 4), (8904, 30), (19303, 0), (12918, 0), (9256, 0), (11855, 20), (18392, 4), (20426, 28), (19402, 21), (14744, 15), (19371, 0), (4645, 2), (83, 9), (10331, 3), (9430, 10), (6143, 25), (18367, 0), (12914, 3)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "issues = find_label_issues(labels, pred_probs, return_indices_ranked_by='self_confidence') \n",
+    "top = 20 \n",
+    "print('Cleanlab found %d potential label issues. ' % len(issues)) \n",
+    "print('The top %d most likely label errors:' % top) \n",
+    "print(str(issues[:top])) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e6e2963",
+   "metadata": {},
+   "source": [
+    "Let's look at the top 20 examples cleanlab thinks are most likely to be incorrectly labeled. We obtain the sentences from the original file to display the word-level token label issues in context. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "639a34de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
+    "# !wget https://data.deepai.org/conll2003.zip && mkdir data \n",
+    "# !unzip conll2003.zip -d data/ && rm conll2003.zip "
+   ]
```

Comment:
Did you mean to include this?  

If so, all downloads of data should be at the very start of this notebook.